packages/design-tokens/src/scss/generated/_typography.scss

```diff
@@ -4,7 +4,7 @@ $body-large-text-bold: (
         font-family: "'Inter', sans-serif",
         font-size: 1.125rem,
         font-style: normal,
-        font-weight: 600,
+        font-weight: NaN,
```

Comment:
OK, this will be probably fixed when the new version in Supernova is released. 

When I check older versions in Supernova the weight is still `bold`, but on the shared draft (the latest stuff but not a released version) the weight is already `700`.